config/config.go

```diff
@@ -51,12 +51,21 @@ type Adapter struct {
 }
 
 type Metrics struct {
+	Influxdb   InfluxMetrics     `mapstructure:"influxdb"`
+	Prometheus PrometheusMetrics `mapstructure:"prometheus"`
+}
+
+type InfluxMetrics struct {
 	Host     string `mapstructure:"host"`
 	Database string `mapstructure:"database"`
 	Username string `mapstructure:"username"`
 	Password string `mapstructure:"password"`
 }
 
+type PrometheusMetrics struct {
+	Enabled string `mapstructure:"enabled"`
```

Comment:
What is this `enabled` string? Should it be a `bool`?

Why does it even exist? Wouldn't you just omit this propertyif you didn't want to use prometheus?

Reply:
Because this is the only element of prometheus. Prometheus uses a pull structure, so we don't need to configure for a particular Promethus instance. But bool is a better choice, yes.

Reply:
Actually... I just realized, this isn't even used anywhere. It's just dead code.

Let's remove it until the PR with Prometheus support.